src/openrct2/core/File.cpp

```diff
@@ -45,39 +45,39 @@ namespace File
         return result && ec.value() == 0;
     }
 
-    bool Delete(std::string_view path)
+    bool Delete(u8string_view path)
     {
         std::error_code ec;
         const auto result = fs::remove(u8path(path), ec);
         return result && ec.value() == 0;
     }
 
-    bool Move(std::string_view srcPath, std::string_view dstPath)
+    bool Move(u8string_view srcPath, u8string_view dstPath)
     {
         std::error_code ec;
         fs::rename(u8path(srcPath), u8path(dstPath), ec);
         return ec.value() == 0;
     }
 
-    std::vector<uint8_t> ReadAllBytes(std::string_view path)
+    std::vector<uint8_t> ReadAllBytes(u8string_view path)
     {
 #if defined(_WIN32) && !defined(__MINGW32__)
         auto pathW = String::ToWideChar(path);
         std::ifstream fs(pathW, std::ios::in | std::ios::binary);
 #else
-        std::ifstream fs(std::string(path), std::ios::in | std::ios::binary);
+        std::ifstream fs(u8string(path), std::ios::in | std::ios::binary);
```

Comment:
we don't need this ifdef anymore since we have the fs::path just create an fs::path from the path and pass it to `ifstream` that will handle the wide char internally.